static/js/gtag.js

```diff
@@ -18,9 +18,17 @@ window.addEventListener('DOMContentLoaded', function () {
     // Events on links should be sent via beacon, see:
     // https://support.google.com/analytics/answer/7478520?hl=en
     if (elem.hasAttribute('href')) {
-      data.event_callback = function() { document.location = elem.href; };
+      var updated = false;
+      var callbackFn = function() {
+        if (updated) return;
+        updated = true;
+        document.location = elem.href;
+      };
+      data.event_callback = callbackFn;
       data.transport_type = 'beacon';
-      e.preventDefault();
+
+      // Fallback location change in case the Google Tag Manager is blocked.
+      setTimeout(callbackFn, 400);
```

Comment:
Can we make this smaller? Just thinking 100ms, must be enough. We actively make the behavior of the site worse...